File: runner/src/integration-tests/java/com/daylifecraft/minigames/DevTest.java
package com.daylifecraft.minigames;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class DevTest {
  @Test
  void testDev() {
    Dev.setDev(false);
    Assertions.assertFalse(Dev.isDev(), "Dev.isDev() should return false");

    Dev.setDev(true);
    Assertions.assertTrue(Dev.isDev(), "Dev.isDev() should return true");
  }
}

File: runner/src/test/java/DevTest.kt
import com.daylifecraft.minigames.Dev
import org.junit.jupiter.api.Test
import kotlin.test.assertFalse
import kotlin.test.assertTrue

internal class DevTest {
  @Test
  fun testDev() {
    Dev.isDev = false
    assertFalse(Dev.isDev, "Dev.isDev() should return false")

    Dev.isDev = true
    assertTrue(Dev.isDev, "Dev.isDev() should return true")
  }
}
